frappe/contacts/report/addresses_and_contacts/addresses_and_contacts.js

```diff
@@ -9,6 +9,13 @@ frappe.query_reports["Addresses And Contacts"] = {
 			"label": __("Entity Type"),
 			"fieldtype": "Link",
 			"options": "DocType",
+			"get_query": function() {
+				return {
+					"filters": {
+						"name": ["in", "Contact, Address"],
```

Comment:
Why? what's your requirement here?

Reply:
Addresses and Contacts report should not show all the available doctypes